frontend/src/components/ccofApplication/group/LicenseUpload.vue

```diff
@@ -89,7 +90,17 @@ export default {
       }
     },
     isLocked() {
-      if (this.unlockLicenseUpload || this.isChangeRequest) {
+      if(this.isChangeRequest){
+        let currentCR = this.userProfileChangeRequests?.filter(el=>el.changeRequestId===this.changeRequestId)[0];
+        if(currentCR.unlockLicenseUpload){
```

Comment:
should we have null check here? just in case, in some odd event, currentCR is null.